docs/release/release-tasks.md

```diff
@@ -82,9 +82,10 @@ As of now we ask for volunteers in Slack and office hours.
 
 #### Finalize release schedule and team
 
-1. Finalize release schedule and team in `release-1.4.md`.
+1. Finalize release schedule and team in `release-x.y.md`.
```

Comment:
I don't mind but just fyi "* The examples in this document are based on the v1.4 release cycle."

The idea was to have concrete examples to make everything easier to understand.

I"m fine either way, but we should probably be consistent and either use a concrete example or use x.y, x.y+1, x.y-1 etc

Reply:
Makes sense to me. Updated.